pom.xml

```diff
@@ -67,7 +67,12 @@
                 <groupId>org.jenkins-ci</groupId>
                 <artifactId>symbol-annotation</artifactId>
                 <version>1.23</version>
-            </dependency>  
+            </dependency>
+            <dependency>
+                <groupId>com.google.errorprone</groupId>
+                <artifactId>error_prone_annotations</artifactId>
+                <version>2.10.0</version>
+            </dependency>
```

Comment:
was excluded for `script-security` do we need to version manage this or should it be excluded _if_ coming from core via guava?